src/omnicore/version.h

```diff
@@ -21,7 +21,7 @@
 #define OMNICORE_VERSION_PATCH       99
 
 // Non-public build number/revision (usually zero)
-#define OMNICORE_VERSION_BUILD       0
+#define OMNICORE_VERSION_BUILD       3
```

Comment:
Why?

Reply:
Ah - this was from the UIT prototype I was providing to the guys - when I was trying to debug I wanted to make sure they were building the right version (but I found the values in `bitcoin-config.h` took precedence so didn't quite work out how I wanted!).

Unnecessary so reversed in f328ef7d3ce72e7c493a60d4262caa8f5b786cd0.